src/createWavelets.jl

```diff
@@ -60,7 +60,6 @@ end
 
 function mother(this::CWT{W,T,Morse,N}, s::Real,
```

Comment:
So the lack of `sWidth` is causing the errors in the tests; even if you don't use it, you should include it in the function header so that `computeWavelets` using `mother` doesn't break. This is probably me having not the best code hygiene, but please bear with me.